### PR TITLE
Added Card, Input, Image component.

### PR DESCRIPTION
**Pull request by Nikhil-Nirmal**

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

File: components/Card/Card.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+.discount{
+    margin-left:0.5rem ;
+    font-weight: 400;
+    font-size: 1rem;
+    color: green;
+    background-color: var(--card-white) ;
+}
+
+button {
```
Comment (muskaanshah) [inline components/Card/Card.css]: Try grabbing elements via class name and not element name

File: components/Card/Card.html [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+                    </li>
+                </ul>
+            </div>
+        </sidebar>
+
+        
+        <main class="heroSection">
+            <h1 class="title"> <highlight class="num">5.&emsp;</highlight> Cards :</h1>
+            <div class="line"></div>
+            <h3 class="description">Badges are small circles, positioned either at top-right or bottom-right of the parent componentA “card” is a UI design pattern that groups related information in a flexible-size container visually resembling a playing card.</h3>
```
Comment (muskaanshah) [inline components/Card/Card.html]: There's a description about badges, I think you forgot to edit that

Comment (Nikhil-Nirmal) [reply components/Card/Card.html]: have added the description

File: components/Card/Card.css (hunk 1 above)
Comment (Nikhil-Nirmal) [reply components/Card/Card.css]: Yes will take note of that

**Review by muskaanshah — APPROVED** [review]

**Pull request merged**